src/applications/mhv-secure-messaging/tests/e2e/secure-messaging-folders.cypress.spec.js

```diff
@@ -13,38 +13,36 @@ describe(manifest.appName, () => {
 
   it('Check the Inbox folder', () => {
     cy.injectAxe();
-    cy.axeCheck(AXE_CONTEXT, {});
+    cy.axeCheck(AXE_CONTEXT);
     FolderLoadPage.verifyFolderHeaderText('Inbox');
-    FolderLoadPage.verifyBackToMessagesButton();
+    FolderLoadPage.verifyBreadCrumbsLength(4);
+    FolderLoadPage.backToFolder('Messages');
   });
 
   it('Check the Draft folder', () => {
     FolderLoadPage.loadDraftMessages();
     cy.injectAxe();
-    cy.axeCheck(AXE_CONTEXT, {});
+    cy.axeCheck(AXE_CONTEXT);
     FolderLoadPage.verifyFolderHeaderText('Drafts');
-    FolderLoadPage.verifyBackToMessagesButton();
+    FolderLoadPage.verifyBreadCrumbsLength(4);
+    FolderLoadPage.backToFolder('Messages');
   });
 
   it('Check the Sent folder', () => {
     FolderLoadPage.loadSentMessages();
     cy.injectAxe();
-    cy.axeCheck(AXE_CONTEXT, {});
+    cy.axeCheck(AXE_CONTEXT);
     FolderLoadPage.verifyFolderHeaderText('Sent');
-    FolderLoadPage.verifyBackToMessagesButton();
+    FolderLoadPage.verifyBreadCrumbsLength(4);
+    FolderLoadPage.backToFolder('Messages');
   });
 
   it('Check the Trash folder', () => {
     FolderLoadPage.loadDeletedMessages();
     cy.injectAxe();
-    cy.axeCheck(AXE_CONTEXT, {
-      rules: {
-        'aria-required-children': {
-          enabled: false,
-        },
-      },
-    });
+    cy.axeCheck(AXE_CONTEXT);
     FolderLoadPage.verifyFolderHeaderText('Trash');
-    FolderLoadPage.verifyBackToMessagesButton();
+    FolderLoadPage.verifyBreadCrumbsLength(4);
+    FolderLoadPage.backToFolder('Messages');
```

Comment:
should we consider including `FolderLoadPage.backToFolder('Messages');` in `afterEach` ?